ansible/roles/baselayout/tasks/partials/repo/centos5.yml

```diff
@@ -14,5 +14,5 @@
 
 - name: install epel
   yum:
-    name: "http://dl.fedoraproject.org/pub/epel/5/{{ ansible_architecture }}/epel-release-5-4.noarch.rpm"
+    name: "https://dl.fedoraproject.org/pub/epel/epel-release-latest-5.noarch.rpm"
```

Comment:
I think this needs to be the archive link:

http://archives.fedoraproject.org/pub/archive/epel/epel-release-latest-5.noarch.rpm